cf/functions.py

```diff
@@ -1275,6 +1557,27 @@ def total_memory():
     return CONSTANTS["TOTAL_MEMORY"]
 
 
+def is_log_level_info(logger):
+    """Return True if and only if log level is at least INFO.
```

Comment:
```suggestion
    """Return True if and only if log level is at least as verbose as INFO.
```

or:

```suggestion
    """Return True if and only if log level is at least INFO i.e. value 20 or less.
```

or similar, ideally, to clarify you mean 'info' or 'debug' levels i.e. more verbose, given the numeric equivalents decrease with increase in verbosity.